example/angular_components_example/build.yaml

```diff
@@ -12,9 +12,6 @@ targets:
         enabled: True
       angular_gallery_section:
         enabled: False
-      build_web_compilers|dart2js_archive_extractor:
```

Comment:
Why these changes? @nshahan ?

Reply:
Looks like a bad merge. I don't intend to remove those. I'll add them back real quick. :smiley: